dockstore-webservice/src/main/java/io/dockstore/webservice/resources/OrganizationResource.java

```diff
@@ -853,6 +854,30 @@ static boolean doesOrganizationExistToUser(Long organizationId, Long userId, Org
     }
 
 
+    static boolean isUserAdminOrMaintainer(Organization organization, Long userId) {
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/gh/dockstore/dockstore/pullRequest?prid=6348842)